Library/Homebrew/manpages.rb

```diff
@@ -3,6 +3,7 @@
 
 require "cli/parser"
 require "erb"
+require "i18n"
```

Comment:
As an aside, line 40 below appears to be the single use of `i18n` in the repo, though I don't have a sense of what it's doing. But if it's not doing any heavy lifting, it might be worth yanking the dependency.

Reply:
@dduugg We should definitely yank it, it's doing nothing of note. I think just try to remove it and see what it does to `brew generate-man-completions` output locally.

Reply:
Looks like it's only affecting the use of oxford commas in generating manpages 😄 

I don't know if someone in a non-en locale should test this too (I don't think it matters though).

I can yank the dependency in a follow-up PR, for now I moved all the i18n code into `manpages`.

Reply:
Interestingly, removing the transliterate step arguably improves the docs, look at the subtle changes to Misty De Méo and Vítor Galvão (it now includes the diacritics):

```diff
 Homebrew\'s Project Leader is Mike McQuaid\.
 .
 .P
-Homebrew\'s Project Leadership Committee is Colin Dean, Issy Long, Jonathan Chang, Mike McQuaid, Misty De Meo, and Sean Molenaar\.
+Homebrew\'s Project Leadership Committee is Colin Dean, Issy Long, Jonathan Chang, Mike McQuaid, Misty De Méo, and Sean Molenaar\.
 .
 .P
 Homebrew\'s Technical Steering Committee is Bo Anderson, FX Coudert, Michka Popoff, Mike McQuaid, and Rylan Polster\.
@@ -3463,7 +3463,7 @@ Homebrew\'s Technical Steering Committee is Bo Anderson, FX Coudert, Michka Popo
 Homebrew\'s maintainers are Alexander Bayandin, Bevan Kay, Bo Anderson, Branch Vincent, Caleb Xu, Carlo Cabrera, Daniel Nachun, Dawid Dziurla, Dustin Rodrigues, Eric Knibbe, FX Coudert, George Adams, Issy Long, Markus Reiter, Miccal Matthews, Michael Cho, Michka Popoff, Mike McQuaid, Nanda H Krishna, Patrick Linnane, Rui Chen, Ruoyu Zhong, Rylan Polster, Sam Ford, Sean Molenaar, and Thierry Moisan\.
 .
 .P
-Former maintainers with significant contributions include Misty De Meo, Shaun Jackman, Vitor Galvao, Claudia Pellegrino, Seeker, William Woodruff, Jan Viljanen, JCount, commitay, Dominyk Tiller, Tim Smith, Baptiste Fontaine, Xu Cheng, Martin Afanasjew, Brett Koonce, Charlie Sharpsteen, Jack Nagel, Adam Vandenberg, Andrew Janke, Alex Dunn, neutric, Tomasz Pajor, Uladzislau Shablinski, Alyssa Ross, ilovezfs, Chongyu Zhu and Homebrew\'s creator: Max Howell\.
+Former maintainers with significant contributions include Misty De Méo, Shaun Jackman, Vítor Galvão, Claudia Pellegrino, Seeker, William Woodruff, Jan Viljanen, JCount, commitay, Dominyk Tiller, Tim Smith, Baptiste Fontaine, Xu Cheng, Martin Afanasjew, Brett Koonce, Charlie Sharpsteen, Jack Nagel, Adam Vandenberg, Andrew Janke, Alex Dunn, neutric, Tomasz Pajor, Uladzislau Shablinski, Alyssa Ross, ilovezfs, Chongyu Zhu and Homebrew\'s creator: Max Howell\.
 .
 .SH "BUGS"
 See our issues on GitHub:
```

The above represents the only change in `brew generate-man-completions`. Given that, I'm ok removing it in this PR.

Reply:
(Although it occurs to me this may be the specific intention, if ASCII-only is still a thing. Will spelunk the code when I get a chance.)

Reply:
Ok, i found the [source](https://github.com/Homebrew/brew/pull/10601), which suggests that removing diacritics is exactly the point. What would you like to do here?

Reply:
There are also a few options [here](https://stackoverflow.com/q/15686752/2193416), if we want to explore non-i18n options for scrubbing diacritics

Reply:
@dduugg I think ASCII seems desirable here unless `man brew` and friends render perfectly for you without it? The simplest middle ground may be to just handle `ãéí` and adding a test that will fail if any additional ones are added/used. Thoughts?

Reply:
`man brew` renders as expected in iTerm2 and Terminal on macOS v12.6.3:

<img width="1263" alt="image" src="https://user-images.githubusercontent.com/697964/224101263-36182128-222b-416d-b96b-02e21be601e2.png">

cc @nandahkrishna, in case he would like to test this branch

Reply:
Sorry for the late reply, indeed they seem to render fine! Thanks!

Reply:
Let's leave/try it as-is! Thanks @dduugg and @nandahkrishna.